internal/fsstat/fsstat_linux.go

```diff
@@ -0,0 +1,45 @@
+package fsstat
+
+import (
+	"log"
+	"os"
+	"syscall"
+
+	tspb "github.com/golang/protobuf/ptypes/timestamp"
+
+	fspb "github.com/google/fswalker/proto/fswalker"
+)
+
+// ToStat returns a fspb.ToStat with the file info from the given file
+func ToStat(info os.FileInfo) *fspb.FileStat {
+	if stat, ok := info.Sys().(*syscall.Stat_t); ok {
+		return &fspb.FileStat{
+			Dev:     stat.Dev,
+			Inode:   stat.Ino,
+			Nlink:   stat.Nlink,
+			Mode:    stat.Mode,
+			Uid:     stat.Uid,
+			Gid:     stat.Gid,
+			Rdev:    stat.Rdev,
+			Size:    stat.Size,
+			Blksize: stat.Blksize,
+			Blocks:  stat.Blocks,
+			Atime: &tspb.Timestamp{
```

Comment:
Would be nice to reuse `timespec2Timestamp(...)` here in the other places below.